src/deploy.js

```diff
@@ -8,12 +9,10 @@ const DELETE_LIMIT = 1000; // https://docs.aws.amazon.com/AWSJavaScriptSDK/lates
 
 const computeCacheControl = (path, cacheControl) => {
 
+  // more specific cache-control rules should come first, since .find() will stop at the first match
   const match = Object.entries(cacheControl).find(([ cacheControlPath ]) => (
-    (cacheControlPath === path)
-    || (
-      cacheControlPath.endsWith('*')
-      && new RegExp(`^${cacheControlPath.replace(/\*?$/, '.*')}$`).test(path)
-    )
+    // prepend forward slash to simulate root directory and enable globs like "/my_root_file.html"
+    micromatch.isMatch(`/${path}`, cacheControlPath)
```

Comment:
Is the leading slash necessary? As `computeCacheControl()` is now applied to local paths it would make sense to expect the user *not* to provide cache control paths starting with a slash. Input paths should be local or remote, not both.

Reply:
Yes, you're correct. I misunderstood how glob matching works.